commons/zenoh-buffers/src/bbuf.rs

```diff
@@ -131,6 +130,7 @@ impl<'a> HasReader for &'a BBuf {
 #[cfg(feature = "test")]
 impl BBuf {
     pub fn rand(len: usize) -> Self {
+        #[allow(unused_imports)]
```

Comment:
Double check if this is really needed.

Reply:
It wasn't anymore. Removed in last commit.